ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1868,17 +1877,22 @@ const RuntimePanel = fp.flow(
       const maxDiskSize = isUsingFreeTierBillingAccount(workspace)
         ? 4000
```

Comment:
I like this idea, apart from this i would like to move the MIN_SIZE and other constants to the constant files. I'll create another PR just to avoid too much noise